src/pages/Admin/Charity/Withdraws/Withdrawer/WithdrawForm/Network/Network.tsx

```diff
@@ -0,0 +1,88 @@
+import { Listbox } from "@headlessui/react";
+import { useEffect } from "react";
+import { useFormContext } from "react-hook-form";
+import { WithdrawValues } from "../types";
+import { useChainsQuery } from "services/apes";
+import Icon, { DrawerIcon } from "components/Icon";
+import QueryLoader from "components/QueryLoader";
+
+export default function Network() {
```

Comment:
before it's not possible to submit transaction when network and destination is mistmatch - now can submit
<img width="570" alt="image" src="https://user-images.githubusercontent.com/89639563/223012669-a30284fc-d77a-440c-974e-eed2fd3b054c.png">

Reply:
Unable to repro. Can you retry this, maybe the issue was fixed with one of the recent commits?

Reply:
@ap-justin **does the issue still occur?**